### PR TITLE
Improvements to Spring Boot Secured booster 

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

File: docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc [hunk 1]
```diff
@@ -50,6 +50,9 @@ $ curl ... | jq '.access_token'
...
 ----
++
+NOTE: When using this result, make sure to remove the surrounding double quotes (`"`).
```
Comment (Ladicek) [inline docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc]: Yea I'm not sure why we aren't using `jq -r`, that doesn't produce the quotes.

**Comment by rhoads-zach** [comment]
[test]

**Comment by rhoads-zach** [comment]
merging for now. @metacosm feel free to address @Ladicek feedback in a follow up PR or file an issue.

**Comment by rhoads-zach** [comment]
merged; 7dd196b3cf82f90cb9c7f8536f1839f573141479

**Comment by Ladicek** [comment]
Using `jq -r` would be nice IMHO -- more straightforward, less admonitions :-)

**Pull request closed without merging**